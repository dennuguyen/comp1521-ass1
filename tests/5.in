11
11
-320