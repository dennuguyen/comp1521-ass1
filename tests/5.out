Enter world size: Enter rule: Enter how many generations: Invalid number of generations
